lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -117,30 +117,26 @@ def test_pdflatex():
 @pytest.mark.style('default')
 @pytest.mark.backend('pgf')
 def test_rcupdate():
-    rc_sets = []
-    rc_sets.append({'font.family': 'sans-serif',
-                    'font.size': 30,
-                    'figure.subplot.left': .2,
-                    'lines.markersize': 10,
-                    'pgf.rcfonts': False,
-                    'pgf.texsystem': 'xelatex'})
-    rc_sets.append({'font.family': 'monospace',
-                    'font.size': 10,
-                    'figure.subplot.left': .1,
-                    'lines.markersize': 20,
-                    'pgf.rcfonts': False,
-                    'pgf.texsystem': 'pdflatex',
-                    'pgf.preamble': ['\\usepackage[utf8x]{inputenc}',
-                                     '\\usepackage[T1]{fontenc}',
-                                     '\\usepackage{sfmath}']})
-    tol = (6, 0)
-    original_params = mpl.rcParams.copy()
+    rc_sets = [{'font.family': 'sans-serif',
```

Comment:
Could actually be a parametrized test. But I don't require it as part of this PR.